extensions/oidc-token-propagation-reactive/deployment/src/main/java/io/quarkus/oidc/token/propagation/reactive/OidcTokenPropagationReactiveBuildTimeConfig.java

```diff
@@ -13,4 +13,18 @@ public class OidcTokenPropagationReactiveBuildTimeConfig {
      */
     @ConfigItem(defaultValue = "true")
     public boolean enabled;
+
+    /**
+     * If the OIDC Token Reactive Propagation is enabled during authentication, such as when you need
```

Comment:
Would it be possible to simplify it a bit, especially the technical part related to Vert.x contexts, for example:
````
If the token propagation is enabled during the security identity augmentation.
<p/>
For example, you may need to use a REST client from {@link io.quarkus.security.identity.SecurityIdentityAugmentor} to propagate the current token to acquire additional roles for the security identity.
<p/> 
Note, enabling this property requires duplicating a Vert.x context to ensure CDI injection is handled safely while the security identity is being augmented.
```

Something along these lines - please tune as you prefer as I thought things like `root context` or a recommendation to confirm it actually works with the integration tests can be difficult to follow - esp the one about the tests - as users may just move past this option due to some uncertainty.

Does it sound reasonable ? I'm not sure if the situation with the root context `/` being used after say `/service` context has been duplicated (if I got it right :-) ) is realistic - but even if it is then it will just not work in this case.

If it is realistic then please give an example of what kind of setup attempting to duplicate the context is not advisable

Reply:
I wonder if inside config properties Javadoc we should also use relative links to the guides like `xref:duplicated-context[this guide]`. I saw that Hibernate config is using xref somewhere, I'll try it.

Reply:
Yeah, that's better, done.